package.json

```diff
@@ -4,25 +4,20 @@
   "description": "A native module for when you just need node to back off for a few milliseconds",
   "keywords": [],
   "main": "index.js",
-  "gypfile": true,
   "dependencies": {
     "nan": "<2.0.0 >1.6.2 || 1.5.0",
     "node-pre-gyp": "^0.6.2"
   },
-  "devDependencies": {},
-  "bundledDependencies": [
-    "node-pre-gyp"
-  ],
```

Comment:
The documentation for node-pre-gyp says you need to do this (https://github.com/mapbox/node-pre-gyp#1-add-new-entries-to-your-packagejson)  I'm not sure why though.